lambda/resolveSingleCellGeoLocation.ts

```diff
@@ -41,7 +40,7 @@ const deviceFetcher = getDeviceAttributesById({ db, DevicesTableName })
 
 const { track, metrics } = metricsForComponent('singleCellGeo')
 
-const trackFetch = loggingFetch({ track, log })
+const trackFetch = validatedLoggingFetch({ track, log })
```

Comment:
```suggestion
const trackFetch = validatedFetch(loggingFetch({ track, log }))
```

Since `loggingFetch` returns a fetch compatible function, `validatedLoggingFetch` should not care about the fact that it deals with a fetch that is augmented with logging capabilites. Instead make it accept a fetch instance as parameter. (Unix command design principle)

Reply:
I will use the existing function, [validatedFetch](https://github.com/hello-nrfcloud/backend/blob/291-implement-nrf-cloud-api-response-validation/nrfcloud/validatedFetch.ts), because

- It accepts `fetch` instance 
- However, I need to modify the function to accept 3rd parameter because the existing function will use `GET` method and `Content-Type` as `application/json` as its defaults.